components/ingest-service/integration_test/project_delete_actions_test.go

```diff
@@ -0,0 +1,93 @@
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	iBackend "github.com/chef/automate/components/ingest-service/backend"
+	"github.com/chef/automate/components/ingest-service/backend/elastic/mappings"
+)
+
+func TestProjectDeleteActions(t *testing.T) {
+	var (
+		ctx = context.Background()
+	)
+
+	cases := []struct {
+		description         string
+		existingProjectIDs  []string
+		projectIDToBeDelete string
+		expectedProjectIDs  []string
+	}{
+		{
+			description:         "Deleting the last project tag",
+			existingProjectIDs:  []string{"target_project"},
+			projectIDToBeDelete: "target_project",
+			expectedProjectIDs:  []string{},
+		},
+		{
+			description:         "Deleting a project that does not exist",
+			existingProjectIDs:  []string{},
+			projectIDToBeDelete: "target_project",
+			expectedProjectIDs:  []string{},
+		},
+		{
+			description:         "Deleting a project on a node with multiple projects",
+			existingProjectIDs:  []string{"project3", "target_project", "project9"},
+			projectIDToBeDelete: "target_project",
+			expectedProjectIDs:  []string{"project3", "project9"},
+		},
+	}
+
+	for _, test := range cases {
+		t.Run(fmt.Sprintf("action match: %s", test.description),
+			func(t *testing.T) {
+				action := iBackend.InternalChefAction{
+					Projects:   test.existingProjectIDs,
+					RecordedAt: time.Now(),
+				}
+				suite.IngestActions([]iBackend.InternalChefAction{action})
+				defer suite.DeleteAllDocuments()
+
+				// Send a project rules update event
+				esJobID, err := suite.ingest.DeleteActionProjectTag(ctx, test.projectIDToBeDelete)
+				require.Nil(t, err)
```

Comment:
Prefer to use `NoError` and add a brief message, e.g.:
```suggestion
				require.NoError(t, err, "rules update event failed")
```